src/Init/Data/Nat/Div.lean

```diff
@@ -203,6 +203,10 @@ theorem mod_add_div (m k : Nat) : m % k + k * (m / k) = m := by
   | base x y h => simp [h]
   | ind x y h IH => simp [h]; rw [Nat.mul_succ, ← Nat.add_assoc, IH, Nat.sub_add_cancel h.2]
 
+theorem mod_def (m k : Nat) : m % k = m - k * (m / k) := by
```

Comment:
Technically this is not the definition for `Nat`, but I like the parallelism with the `Int` naming, and will worry about whether this needs to be changed later.